src/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env"]
```

Comment:
@SamTV12345  this file 🤗

Reply:
Thanks for the end. Good question. Now with SWC this shouldn't be needed anymore.

Reply:
> Now with SWC this shouldn't be needed anymore.

normally yesss